test/test-app/src/app/test-strict-null-checks/test-strict-null-checks.component.ts

```diff
@@ -0,0 +1,104 @@
+/*
+    Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-test-strict-null-checks',
+  templateUrl: './test-strict-null-checks.component.html',
+  styleUrls: ['./test-strict-null-checks.component.css'],
+})
+export class TestStrictNullChecksComponent implements OnInit {
+  constructor() {}
+
```

Comment:
let's add a class member variable(which does not have type undefined.) and assign it to null/undefined